src/Settings/Builders/Common.hs

```diff
@@ -46,13 +49,19 @@ cWarnings =
             , notM (flag GccIsClang) ? notM windowsHost ? arg "-Werror=unused-but-set-variable"
             , notM (flag GccIsClang) ? arg "-Wno-error=inline" ]
 
+packageDatabaseArgs :: Args
+packageDatabaseArgs = do
+    stage  <- getStage
+    dbPath <- expr $ packageDbPath stage
+    expr $ need [dbPath -/- packageDbStamp]
+    top    <- expr topDirectory
+    root   <- getBuildRoot
+    prefix <- ifM (builder Ghc) (return "-package-db ") (return "--package-db=")
+    arg $ prefix ++ top -/- root -/- inplacePackageDbPath stage
+
 bootPackageDatabaseArgs :: Args
 bootPackageDatabaseArgs = do
     stage  <- getStage
     dbPath <- expr $ packageDbPath stage
     expr $ need [dbPath -/- packageDbStamp]
```

Comment:
Is this still needed? Shouldn't `stage0 ? packageDatabaseArgs` be sufficient?

Reply:
YES!

Reply:
Yes, `stage0 ? packageDatabaseArgs` should be sufficient.